### DISCUSSION TITLE
Support for reporting of sub-band CQI

### DISCUSSION THREAD

**Discussion by rolandmartinson7890** [srsran/srsRAN_Project#440] [General Help] [answered]
Good day srsRAN users. I would like to ask if there is a plan to report sub-band CQI in srsRAN Project?

**Comment by herlesupreeth** [comment, marked as answer]
Please refer to the discussions here - https://github.com/srsran/srsRAN_Project/discussions/329 and https://github.com/srsran/srsRAN_Project/discussions/320

**Reply by rolandmartinson7890** [reply]
Thanks @herlesupreeth
